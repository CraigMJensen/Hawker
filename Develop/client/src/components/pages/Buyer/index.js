import React from 'react';

function Buyer() {
  return (
    <div>
      <h2>Buyer</h2>
    </div>
  );
}

export default Buyer;
