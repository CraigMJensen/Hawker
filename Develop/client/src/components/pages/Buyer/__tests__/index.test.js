// Import dependencies
import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Buyer from '..';

// After each test, make sure there isnt any leftover memory that could give you false results
afterEach(cleanup);

// Declare the components you are testing
describe('Buyer component', () => {
  // verify the component is rendering properly
  it('Buyer component renders', () => {
    render(<Buyer />);
  });

  // compare snapshot versions of the DOM node structure
  it('matches snapshot DOM node structure', () => {
    const { asFragment } = render(<Buyer />);
    expect(asFragment()).toMatchSnapshot();
  });
});