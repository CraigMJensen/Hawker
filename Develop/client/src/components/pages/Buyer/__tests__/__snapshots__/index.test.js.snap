// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buyer component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div>
    <h2>
      Buyer
    </h2>
  </div>
</DocumentFragment>
`;
